src/LevelManager.hh

```diff
@@ -74,7 +74,7 @@ namespace ignition
     ///   when the level is reloaded. Likewise, they should not be deleted.
     /// * Entities spawned during simulation are part of the default level.
     ///
-    class LevelManager
+    class IGNITION_GAZEBO_VISIBLE LevelManager
```

Comment:
This isn't installed, so I think it shouldn't be needed.

Reply:
same that in https://github.com/ignitionrobotics/ign-gazebo/pull/563#discussion_r560574124